src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerGetTranscriberTranscriptsIntTest.java

```diff
@@ -246,40 +260,23 @@ void getTranscriberTranscriptsShouldReturnTranscriptRequestsWithTranscriberAndCo
 
         final MvcResult mvcResult = mockMvc.perform(requestBuilder)
             .andExpect(status().isOk())
+            .andExpect(jsonPath("$.[*].transcription_id").isArray())
```

Comment:
Can we make it clear in the test title the scenario this is actually trying to test? Alot of the tests in this class start with the same 'getTranscriberTranscripts'. It's difficult to easily make out if we have covered every scenario. 

Can you confirm there is a test scenario to cover when the transcription was hidden on day 1 and then the request is made on day 2? I can't see one in that file.

Reply:
Done in commit 0fd4bfde. Yes that test was showing the 2 "assigned" transcription states with one record being filtered. Renamed and added a comment to make it more obvious.

* shouldReturnTranscriptRequestsWithTranscriberAndCompletedTodayStatusOk
This test expects that this "Complete" (trs_id=6) transcription (tra_id=101) is hidden from "Your work > Completed today" view because the workflow_ts is BEFORE TODAY (workflow_ts='2023-11-24 12:53:42.839577+00').